CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+0. [Contribution review process](#contribution-review-process)
+0. [Community](#community)
+
+## Found a bug?
+Please don't hesitate to open an [issue](https://github.com/trolster/ur-cli/issues). We'll need to know the following:
+
+* Version of Node
+* Platform you're running on (macOS, SunOS, Linux, Windows)
+* Architecture you're running on (32bit or 64bit and x86 or ARM)
+* If you are getting any error messages, please include them as well
+
+## Types of contributions we're looking for
+First and foremost, this software is supposed to be easy to set up and use. Your first contribution might be starting a new conversation, or adding to an existing conversation, around making things easier to understand and/or easier to use. You can do so under [Issues](https://github.com/trolster/ur-cli/issues).
+
+There are also many ways you can directly contribute to the cli (in descending order of need):
+
+* Writing tests.
+* Add examples to the documentation that help people understand how to use the cli and how to contribute.
+* Revise language to be more approachable and friendly.
+* Translate docs into other languages.
+* Propose a new feature (open an issue to discuss before writing any code!).
+
+Interested in making a contribution? Read on!
+
+## Ground rules & expectations
+
+Before we get started, here are a few things we expect from you (and that you should expect from others):
+
+* Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and projects, which means we likely have different perspectives on "how coding is done." Try to listen to others rather than convince them that your way is correct.
+* If you open a pull request, please ensure that your contribution passes all tests. If there are test failures, you will need to address them before we can merge your contribution.
+
+## How to contribute
+
+If you'd like to contribute, start by searching through the [issues](https://github.com/trolster/ur-cli/issues) and [pull requests](https://github.com/trolster/ur-cli/pulls) to see whether someone else has raised a similar idea or question.
+
+If you don't see your idea listed, and you think it fits into the goals of this guide, do one of the following:
+* **If your contribution is minor,** such as a typo fix, **or self-contained,** such as writing a translation, open a pull request.
+* **If your contribution is major,** such as a new feature, start by opening an issue first. That way, other people can weigh in on the discussion before you do any work.
+
+## Style guide
+If you're writing code, see the [Airbnb style guide](https://github.com/airbnb/javascript) to help your style match the rest of the code.
+
+## Setting up your environment
+
+This software runs in Node.js. If you have [Node.js](https://nodejs.org/en/) version 6 or higher installed you are good to go.
+
+Node.js install guide: https://docs.npmjs.com/getting-started/installing-node
```

Comment:
What about:

To get started with Node, check out:
- [Node.js install guide](https://docs.npmjs.com/getting-started/installing-node)
- [The npm package manager guide](https://docs.npmjs.com/getting-started/what-is-npm)